File: Hurb/Hurb/Model/Pagination.swift
//
//  Pagination.swift
//  Hurb
//
//  Created by Alexandre Papanis on 08/08/19.
//  Copyright © 2019 Papanis. All rights reserved.
//

import Foundation

struct Pagination: Codable {
    let count: Int?
    let firstPage: String?
    let nextPage: String?
    let previousPage: String?
    let lastPage: String?
    
}

File: ChallengeAlpha/app/src/main/java/com/hurb/challengealpha/model/Pagination.kt
package com.hurb.challengealpha.model

import com.google.gson.annotations.SerializedName

data class Pagination(

    @SerializedName("count") val count: Int,
    @SerializedName("firstPage") val firstPage: String,
    @SerializedName("nextPage") val nextPage: String,
    @SerializedName("previousPage") val previousPage: String,
    @SerializedName("lastPage") val lastPage: String
)